tests/backends/aiida_django/migrations/test_migrations_0046_inline_calc_attributes.py

```diff
@@ -0,0 +1,98 @@
+# -*- coding: utf-8 -*-
+###########################################################################
+# Copyright (c), The AiiDA team. All rights reserved.                     #
+# This file is part of the AiiDA code.                                    #
+#                                                                         #
+# The code is hosted on GitHub at https://github.com/aiidateam/aiida-core #
+# For further information on the license, see the LICENSE.txt file        #
+# For further information please visit http://www.aiida.net               #
+###########################################################################
+# pylint: disable=import-error,no-name-in-module,invalid-name
+"""Test the migration that updates the attributes of InlineFunctions that have been transformed into CalcFunctions."""
+from aiida.backends.general.migrations import utils
+from .test_migrations_common import TestMigrations
+
+
+class TestCalcFunctionAttributesMigration(TestMigrations):
+    """Test the migration for the attributes of InlineFunctions that have been transformed into CalcFunctions."""
+
+    migrate_from = '0045_dbgroup_extras'
+    migrate_to = '0046_inline_calc_attributes'
+
+    @staticmethod
+    def get_source_code(full_file=False):
+        """Returns a string containing the source code for the calcfunction"""
+
+        # yapf: disable
+        text_i = ('from aiida import orm\n'
+                  'from aiida.engine import calcfunction\n'
+        )
+        text_t = ('@calcfunction\n'
+                  'def sum_inline( int1, int2 ):\n'
+                  '    return int1 + int2\n'
+        )
+        text_f = ('num1 = orm.Int(2)\n'
+                  'num2 = orm.Int(3)\n'
+                  'num3 = sum_inline( int1, int2 )\n'
+                  'print(num3)\n'
+        )
+        # yapf: enable
+
+        if full_file:
+            text_t = text_i + '\n' + text_t + '\n' + text_f
+
+        return text_t
+
+    def setUpBeforeMigration(self):
+        calcnode = self.DbNode(
```

Comment:
I would another `CalcFunctionNode` but one that does not corresponds to a legacy node that was migrated. This to test that these nodes are not affected. You could for example set its exit status to a non-zero value and check after migration that it still is non-zero. If the filtering in your migration were to be incorrect, this would have been set to zero.